scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -292,19 +292,30 @@ object H2OMOJOCache extends H2OMOJOBaseCache[EasyPredictModelWrapper, H2OMOJOMod
 
   private def canGenerateContributions(model: GenModel): Boolean = {
     model match {
-      case _: PredictContributionsFactory =>
+      case m: PredictContributionsFactory =>
         val modelCategory = model.getModelCategory
-        modelCategory == ModelCategory.Regression || modelCategory == ModelCategory.Binomial
-      case _ => false
+        if (modelCategory != ModelCategory.Regression && modelCategory != ModelCategory.Binomial) {
+          throw new IllegalArgumentException(s"""
+              | Computing contributions on MOJO of type '${m.getModelCategory}' is only supported for regression
+              | and binomial model categories!
+              | The column 'contributions' will not be generated."
```

Comment:
I would delete this sentence. In case of the exceptaion, nothing will be generated.

Reply:
Good point, will do